### DISCUSSION TITLE
Any news about supporting GitHub Sponsors on Brazil? :brazil:

### DISCUSSION THREAD

**Discussion by andreynering** [community/community#16921] [Other Feature Feedback, Questions, & Ideas] [answered]
I'd just like to say that, despite the 3 year wait, I'm still interested in being able to use GitHub  Sponsors (and I believe many other Brazilians are as well).

I still like that, in the meantime, we can at list link to external platforms like Open Collective. Thanks for that!

**Comment by jlord** [comment, marked as answer]
I'm excited to say that as of today we're live in Brazil!!! https://github.blog/2022-06-01-github-sponsors-launches-in-brazil/ 🎉

**Reply by andreynering** [reply]
Hi @jlord, thanks a lot for this! ❤️ 

While trying to setup my Stripe account, I don't see Brazil in the list of countries in the Stripe website:

![Screen Shot 2022-06-01 at 11 14 46](https://user-images.githubusercontent.com/7011819/171426049-7a2a3630-5e74-4b52-9c5a-30a2308dfdff.png)

**Reply by andreynering** [reply]
Nevermind, it appeared after some minutes

**Reply by fabiooshiro** [reply]
I'm from Brazil and I have 25 bucks on stripe.
How can I receive the money?
